rundeck-scripts/start-node

```diff
@@ -167,6 +167,13 @@ while (( i )); do
 	if [[ -n "$container_id" ]]; then
 		echo "RNode is running"
 		nohup docker logs -f $container_id &> $DIAG_DIR/console.log &
+
+                node_pid="$(docker inspect -f '{{.State.Pid}}' rnode || echo 0)"
+                if (( node_pid == 0 )); then
+                        echo "Could not get node PID"
+                        break
+                fi
+                nohup ./pmap.py "$node_pid" "perf-bootstrap.c.developer-222401.internal" 8091 testnet1 &
```

Comment:
I _think_ that with this `nohup` you should also `>/dev/null 2>&1` (or `&>/dev/null`) or else the forked process will stay connected to `start-node`'s stdout/stderr, which can cause `pmap.py` getting `SIGPIPE` for example when it prints something on stderr (an exception) but the parent stderr will be closed already (after `start-node` finishes). If the unintended output here is worth keeping, then replace `/dev/null` with e.g. `/var/lib/rnode-diag/current/pmap.log`. But I don't think it's neccessary.